packages/front-end/services/RadixTheme.tsx

```diff
@@ -14,7 +14,7 @@ export const RadixTheme: FC<RadixThemeProps> = ({ children, flip }) => {
     <Theme
       accentColor="violet"
       panelBackground="solid"
-      appearance={computedTheme}
+      {...(flip ? { appearance: computedTheme } : {})}
```

Comment:
This is an important change, where before we were explicitly passing `appearance` which would default to `light` if localStorage was unavailable, now we rely on the html class `light-theme` and `dark-theme` that the Snippet manages.